packages/hierarchy-browser/src/common/dataProviders/CommunityDataProvider.ts

```diff
@@ -132,16 +127,20 @@ export class CommunityDataProvider {
 	public get communityId(): CommunityId {
 		return this._community
 	}
-
+	// <--- Getters/Setters --->
```

Comment:
This may be better expressed with 
// #region Getters/Setters
<your code>
// #endregion

Then you can use code folding in vscode to collapse that section